logging.md

```diff
@@ -0,0 +1,34 @@
+# Conventions for LOGGING
+[See issues](https://github.com/OCR-D/spec/issues/11)
+All logging may be stored in files. 
+If so, they must be given in Provenance as a result. 
+
+## Log Levels
+(Is there any definition we could link to)
```

Comment:
Unfortunately I am not aware of any...[RFC 5424](https://tools.ietf.org/html/rfc5424) is not super verbose on this and the best explanation I could find is from [Stackoverflow](https://stackoverflow.com/a/5278006/1919465).